lib/tape.js

```diff
@@ -0,0 +1,286 @@
+/**
+ * Codemod for transforming Tape tests into AVA.
+ */
+
+const tapeToAvaAsserts = {
+	fail: 'fail',
+	pass: 'pass',
+
+	ok: 'truthy',
+	true: 'truthy',
+	assert: 'truthy',
+
+	notOk: 'falsy',
+	false: 'falsy',
+	notok: 'falsy',
+
+	error: 'ifError',
+	ifErr: 'ifError',
+	iferror: 'ifError',
+
+	equal: 'is',
+	equals: 'is',
+	isEqual: 'is',
+	strictEqual: 'is',
+	strictEquals: 'is',
+
+	notEqual: 'not',
+	notStrictEqual: 'not',
+	notStrictEquals: 'not',
+	isNotEqual: 'not',
+	doesNotEqual: 'not',
+	isInequal: 'not',
+
+	deepEqual: 'deepEqual',
+	isEquivalent: 'deepEqual',
+	same: 'deepEqual',
+
+	notDeepEqual: 'notDeepEqual',
+	notEquivalent: 'notDeepEqual',
+	notDeeply: 'notDeepEqual',
+	notSame: 'notDeepEqual',
+	isNotDeepEqual: 'notDeepEqual',
+	isNotEquivalent: 'notDeepEqual',
+	isInequivalent: 'notDeepEqual',
+
+	skip: 'skip',
+	throws: 'throws',
+	doesNotThrow: 'notThrows'
+};
+
+const unsupportedTestFunctions = new Set([
+	// No equivalent in AVA:
+	'timeoutAfter',
+	'deepLooseEqual',
+	'looseEqual',
+	'looseEquals',
+	'notDeepLooseEqual',
+	'notLooseEqual',
+	'notLooseEquals'
```

Comment:
> For all the loose/notLoose ones?

Yes

> BTW do you prefer throwing or console.warn for errors where we fail to concert? If the we only warn, should we leave the file untouched?

We should never throw. We should do as many transformations as we can and warn the ones we couldn't do, preferably with a line number, so users can quickly check and manually do it themselves.